### ISSUE TITLE
Changin the sampling rate

### ISSUE THREAD

**Issue by cnamuangtoun** [mindslab-ai/assem-vc#31]
Hi, I have a voice dataset that is sampled at 16kHz, I saw that inside the cofig file there is specific instruction not to change the audio part of the config including the sampling rate. Is there a way for me to adapt a different sampling rate into the code?

**Comment by wookladin** [comment]
Hi, you should change the configurations of the audio and mel spectrogram.
https://github.com/mindslab-ai/assem-vc/blob/master/config/global/default.yaml#L11-L18

You should also change the configurations in HiFi-GAN.
https://github.com/wookladin/hifi-gan/blob/7395e41169e6549f62f941959675cc1fe7bf9735/config_v1.json#L18-L27

However, if you want to use HiFi-GAN with different configs, you need pre-trained weights of HiFi-GAN with those configs.
Since it takes too many times to train HiFi-GAN from scratch (almost 2 weeks needed), I recommend you to just upsample your dataset to a 22k sampling rate to follow our configurations.

Thanks!

**Comment by cnamuangtoun** [comment]
Thak you for the suggestions! I see that in the model, you are using f0 pitch contour to help the decoder similar to fastspeech2's approach. However in fastspeech2, they showed that using energy along with f0 further helps with improve the MOS of the TTS output. Do you think that including energy into the decoder would help in VC case?

**Comment by wookladin** [comment]
Yes, I've done some additional experiments and found that including energy on the decoder input slightly lowers the noise of the output.
